governance/polls/Budget Ratification Poll - Modify Development and UX Core Unit (DUX-001) Budget (MIP40c3-SP82) - November 14, 2022.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Budget Ratification Poll for Modify Development and UX Core Unit (DUX-001) Budget (MIP40c3-SP82) - November 14, 2022
+summary: MIP40c3-SP82 renews the Development & UX Core Unit (DUX-001) budget from February 1st 2023 through January 31st 2024.
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp82-development-ux-core-unit-budget-dux-001/17611
+parameters:
+  input_format:
+    type: rank-free
+    abstain: [0]
+  victory_conditions:
+    - {
+        type: 'and',
+        conditions: [
+          { type : instant-runoff },
+          { type : comparison, comparator : '>=', value: 10000 }
+        ]
+      }
+    - { type : default, value : 3 }
+  result_display: instant-runoff-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Approve existing budget only (Business as usual)
+   2: Approve reduced budget (NewBase - ~86% of existing budget)
```

Comment:
We're not being consistent with the budget options (here and elsewhere): Sometimes we use percentages, sometimes we use amounts of dai, sometimes we use nothing at all. @prose11 , what do you suggest?

Reply:
Don't love using ~ in the options, not sure if it will mess things up but feels like an easy way to stir up controversy. I think we have the exact amount in DAI and it's easy to express so we should use that when able

Reply:
fully agree that DAI amounts should be used